File: go/1101.go
package main

import "fmt"

func main() {
  var m, n, maior, menor, sum int
  for {
    fmt.Scanf("%d", &m)
    fmt.Scanf("%d", &n)
    if n <= 0 || m <= 0 {
      break
    }

    maior = m
    menor = n
    if n > m {
      maior = n
      menor = m
    }

    sum = 0
    for i := menor; i <= maior; i++ {
      sum += i
      fmt.Printf("%d ", i)
    }

    fmt.Printf("Sum=%d\n", sum)
  }
}

File: lua/1101.lua
while true do
    local m = io.read("*n")
    local n = io.read("*n")

    if n <= 0 or m <= 0 then
        break
    end

    local maior, menor = m, n
    if n > m then
        maior, menor = n, m
    end

    local sum = 0
    for i = menor, maior do
        sum = sum + i
        io.write(string.format("%d ", i))
    end

    print(string.format("Sum=%d", sum))
end
